custom_components/powercalc/data/osram/LIGHTIFY Plug 01/model.json

```
@@ -13,7 +13,8 @@
 	"fixed_config": {
 		"power": 1.04
 	},
-    "aliases": [
-        "Plug 01"
-    ]
+	"aliases": [
+		"Plug 01",
+		"Smart+ plug (AB3257001NJ)"
```

Only the model between parenthesis should be enough.
The alias mechanism tries to get the model this way.

```suggestion
		"AB3257001NJ"
```